API.md

```diff
@@ -0,0 +1,68 @@
+# VMCK API
+
+All API calls require authentication, TBD.
+
+## Methods
+
+### `GET /v1/`
+Returns general information about the service.
+
+Reponse:
+```json
+{
+  "version": "0.0.1"
+}
+```
+
+### `PUT /v1/jobs/source`
+Upload source file for a future job. Returns a new ID.
+
+Request body should be the source file.
+
+Reponse:
+```json
+{"id": "$source_id", "size": 1337}
+```
+
+### `POST /v1/jobs`
+Creates a new job.
+
+Request:
+```json
+{
+  "image": "https://repository.grid.pub.ro/cs/so/linux-2019/so-ubuntu-18-04.ova",
```

Comment:
add a type parameter with the type of the image {"vm", "docker", ...,}
```suggestion
  "type": "vm|docker",
```

Reply:
I've added a new section, `executor`, to encapsulate this information.